### PR TITLE
refactor(oui-icons): keep only woff & woff2 on generated fonts

### PR DESCRIPTION
**Pull request by NicoPennec**
### Description of the Change

On build, don't generate the legacy web fonts (`eot`, `svg`, `ttf`) and example files (`html`, `css`).

### Benefits

Be consistent with the current font support policy (woff & woff2 only).

Reduce the size of the `ui-kit` build (~80KB)

### CODE REVIEW

**Comment by antleblanc** [comment]
Hi @NicoPennec!

Thanks a lot for the optimization you've done 💉.

In order to be able to merge your request, could you please resolve existing conflicts?

**Comment by NicoPennec** [comment]
@antleblanc you right! therefore I can keep the "html" file only for that purpose.

**Comment by jleveugle** [comment]
Hello @NicoPennec,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by antleblanc** [comment]
ping @AxelPeter 😃

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "build:less": "lessc --remcalc packages/oui/stylekit.less dist/oui.css",
     "build:less-min": "lessc --remcalc --clean-css packages/oui/stylekit.less dist/oui.min.css",
     "build:icons": "mkdirp dist/icons && npm run build:icons-generator && node scripts/generate-icons-indexes.js",
-    "build:icons-generator": "icon-font-generator packages/oui-icons/svg/*.svg -p oui-icon -o dist/icons -t span",
+    "build:icons-generator": "icon-font-generator packages/oui-icons/svg/*.svg -p oui-icon -o dist/icons -t span --types 'woff, woff2' --css false --html true",
```
Comment (AxelPeter) [inline package.json]: Why removing the CSS, the HTML demo is useless without it ?

Comment (NicoPennec) [reply package.json]: I will revert and keep the full doc (html +css)

**Pull request merged**